cogs/cmd_interface/profile.py

```diff
@@ -63,7 +63,7 @@ async def _profile(self, interaction: discord.Interaction, member: Optional[disc
                 inline = False
             )
             embed.set_author(
-                name = f"LeetCode profile for {interaction.user}",
+                name = f"LeetCode profile for {result['leetcodeUsername']}",
```

Comment:
Is there anyway to display the discord username instead? 

Hint: From result['discordId'] to get the discord username using discord API.